### DISCUSSION TITLE
What's the different between Laplique's RC LIF and 1st-order LIF?

### DISCUSSION THREAD

**Discussion by AreUTheDoctor** [jeshraghian/snntorch#103] [Q&A] [answered]
Cus I remember Laplique's RC LIF is the general LIF model that everyone is familiar and love but then what's the 1st-order LIF model?

**Comment by jeshraghian** [comment, marked as answer]
They both include one state variable (membrane potential), so qualitatively, they look very similar. This means there will likely be no major difference in a neural net simulation. 

Specifically, Lapicque's RC LIF is parameterized by RC whereas LIF is parameterized by `beta` to reduce the number of hyperparameters.
Lapicque's neuron has the option to define `beta` as well, but `beta` is then used to calculate RC (by assuming C=1). 

Ultimately, if you want to simulate something physically plausible (i.e., an RC circuit) then use Lapicque.
If you want to train a neural net, and have less free parameters to think about, Leaky is best.

The documentation has specifics:

Lapicque: https://snntorch.readthedocs.io/en/latest/snn.neurons_lapicque.html
Leaky: https://snntorch.readthedocs.io/en/latest/snn.neurons_leaky.html
